skimage/draw/draw3d.py

```diff
@@ -2,22 +2,81 @@
 from scipy.special import elliprg
 
 
-def ellipsoid(a, b, c, spacing=(1.0, 1.0, 1.0), levelset=False):
-    """Generate ellipsoid for given semi-axis lengths.
+def _create_rotation_matrix(rotation: tuple[float, float, float]) -> np.ndarray:
```

Comment:
Why not use the scipy https://docs.scipy.org/doc/scipy/reference/generated/scipy.spatial.transform.Rotation.from_euler.html to create the rotation matrix?

Reply:
With regards to using SciPy's `Rotation.from_euler` or our own `EuclideanTransform`, I concur that we generally prefer reusing code even if the definitions differ a bit.

I feel like using `EuclideanTransform` — if possible — would be the way to go here. Though, I'm not very firm on the mathematics, here. I'm curious what you think would be the best way to address this?

Also, would it make sense to replace 

https://github.com/scikit-image/scikit-image/blob/fe57b47ba46e9e11c626b97a8a24cd4d2c41fba1/skimage/transform/_geometric.py#L1228

with SciPy's `Rotation.from_euler` in a separate PR?